code/modules/mob/living/carbon/carbon.dm

```diff
@@ -286,7 +286,7 @@
 						src << "<span class='warning'>You didn't choose anything.</span>"
 						return
 
-					if(!do_after(src, 45))
+					if(!do_after(src, 45,,0))

```

Comment:
the other method is just as likely to cause runtimes if somebody changes the arg name for some reason.